modules/govuk/spec/classes/govuk_nodes_spec_optional.rb

```diff
@@ -53,6 +54,30 @@
 
     let(:hiera_config) { temporary_hiera_file.path }
 
+
+    # The tests for ci_agent class requires the mount variable to be set.
+    # Before migration to AWS, ci_agent hiera was accessible under hiera/class
+    # but now resides under hiera_aws/class/integration and is not accessible since
+    # the tests are using environment vagrant rather than integration.
+    if class_name != "ci_agent"
+      default_mount = {}
```

Comment:
nit: Should this be a string `"{}"`? Seems surprising that the type of the variable is different between the two branches of the conditional.

Reply:
it is a hash actually. Sorry I have been experimenting. I will ping you once it is ready, maybe I should I have put it into draft.

Reply:
Fix now, if you could have another look @sengi . thanks!